docs: add link to wiki page

nfischer opened the pull request:
Pretty trivial change, just wanted to see what @ariporad thinks of it. It gives a shoutout to the two most popular projects in the wiki.

ariporad commented on README.md:
```diff
@@ -23,7 +23,10 @@ The project is [unit-tested](http://travis-ci.org/shelljs/shelljs) and battled-t
-For a non-programmatic implementation of Unix shell commands on top of Node.js, check out [Cash](https://github.com/dthree/cash).
+Think ShellJS is cool? Check out some related projects (like
+[cash](https://github.com/dthree/cash)--a javascript-based POSIX shell--or
+[shx](https://github.com/shelljs/shx)--ShellJS functions exposed straight to
```
Two things:
1. I think the wiki page should be a separate item.
2. I feel like `shx` should get it's own section. Maybe with a little intro.

nfischer replied:
@ariporad could you rewrite this PR to clarify what you mean?

ariporad replied:
Sure.

ariporad replied:
@nfischer: Done. (I made it a separate commit so you could compare the changes, I'll squash them if you like them).

nfischer commented:
@ariporad I rebased this. I changed the wording a little bit. I also removed the docs for the `make` command, and moved these to the wiki (and a link is provided). The rationale behind this is that I want less junk in the way of the command reference, which is probably the most important part of the README.

ariporad commented:
LGTM!
